packages/react-native-renderer/src/ReactNativeEventEmitter.js

```diff
@@ -25,7 +25,7 @@ export {getListener, registrationNameModules as registrationNames};
  */
 
 // Shared default empty native event - conserve memory.
-const EMPTY_NATIVE_EVENT = {};
+const EMPTY_NATIVE_EVENT = (({}: any): AnyNativeEvent);
```

Comment:
This is a bit shady. I actually don't know why it's necessary at all. Native events don't always have event objects?

Reply:
not that I've ever seen

Reply:
(This is specific to React Native)